Dockerfile.sparsify

```diff
@@ -1,9 +1,18 @@
 FROM ubuntu:24.04
 
+ARG UNAME
+ARG UID
+ARG GID
+
 ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update && \
     apt-get install -y android-sdk-libsparse-utils && \
     rm -rf /var/lib/apt/lists/*
 
+RUN userdel -r `getent passwd ${UID} | cut -d : -f 1` > /dev/null 2>&1; \
```

Comment:
should not be done if UID is 0 (root) - just as a fail safe - e.g. https://github.com/commaai/agnos-builder/pull/247/commits/553b9b81ad7eea30321cf772d8a775cf3717e293

Reply:
Not sure if I get your motivation here correctly. But yes its not necessary since you would delete and recreate the same root user. Are there any other issues that could happen?

Reply:
I tried running it with sudo e.g. `sudo build_system.sh` and it failed with `useradd: user 'root' already exists` - probably userdel didn't work for root (the message was silenced and didn't check)